src/lib.rs

```diff
@@ -1565,22 +1565,14 @@ from_t!(F8E4M3);
 from_t!(F8E5M2);
 
 #[cfg(feature = "cuda")]
-unsafe impl cudarc::driver::DeviceRepr for F8E4M3 {
-    fn as_kernel_param(&self) -> *mut std::ffi::c_void {
-        use std::ptr::addr_of;
-        addr_of!(self.0) as *const u8 as *mut _
-    }
-}
+unsafe impl cudarc::driver::DeviceRepr for F8E4M3 {}
```

Comment:
Unsure why cudarc removed `as_kernel_param`, could not find any documentation about that change. It was part of a much larger code reorganization.